lib/pbench/test/unit/agent/task/test_generate_token.py

```diff
@@ -0,0 +1,167 @@
+from click.testing import CliRunner
+import requests
+import responses
+
+from pbench.cli.agent.commands import generate_token
+
+
+class TestGenerateToken:
+
+    USER_SWITCH = "--username"
+    PSWD_SWITCH = "--password"
+    USER_PROMPT = "Username: "
+    PSWD_PROMPT = "Password: "
+    USER_TEXT = "test_user"
+    PSWD_TEXT = "password"
+    TOKEN_TEXT = "what is a token but 139 characters of gibberish"
```

Comment:
Therefore, by definition, this `TOKEN_TEXT` is not a token???

Reply:
I thought about trying to come up with a phrase containing exactly 139 characters, but I found something better to do....

[If your comment was something other than a jest, please un-resolve this "conversation" and elaborate.  :-) ]